robot/rover/comms_demo/comms_demo.ino

```diff
@@ -1,29 +1,28 @@
-// USB : Debug, UART : Production
-#define USB
+#include "Arduino.h"
```

Comment:
also this :')

Reply:
random leftover test file that someone created a long ass time ago